src/redux/actions/profile/profileAction.test.js

```diff
@@ -0,0 +1,28 @@
+import * as profileActions from './profileAction';
+
+describe('thunk middleware function', () => {
+  it('should return pending action type', () => {
+    const actionType = 'FETCH_PROFILE_PENDING';
+    expect(profileActions.fetchProfilePending().type).toEqual(actionType);
+  });
+  it('should return success action type', () => {
+    const actionType = 'FETCH_PROFILE_SUCCESS';
+    expect(profileActions.fetchProfileSuccess().type).toEqual(actionType);
```

Comment:
I think this will be well tested if you test the whole action not only the type. Just see if you can modify these tests.

Reply:
Let me work on this as well.